app/api/api.rb

```diff
@@ -1,8 +1,21 @@
+require 'error_handler'
+
 class API < Grape::API
   format :json
   prefix :api
   version 'v1', :path
 
+  use ErrorHandler
+
+  NON_AUTHENTICATION_PATHS = [
+    '/api/v1/sessions',
+    '/api/v1/registrations'
+  ].freeze
```

Comment:
could you pls review how to skip callbacks in controllers directly, but not in a general one:

there're some `skip_*` callbacks in rails => [documentation](https://api.rubyonrails.org/v6.1.4/classes/AbstractController/Callbacks/ClassMethods.html)